sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -406,6 +407,31 @@ protected static boolean isQuotedSubscript(QualifiedNameReference node) {
         return SUBSCRIPT_LITERAL_PATTERN.matcher(node.toString()).matches();
     }
 
+    @Nullable @VisibleForTesting
```

Comment:
ok it's fine, but pls use new lines for each annotation, better for readability ;)